src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/ChefExtensionTests.cs

```diff
@@ -0,0 +1,34 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Xunit;
+
+namespace Microsoft.Azure.Commands.Compute.Test.ScenarioTests
+{
+    public class ChefExtensionTests
+    {
+        public ChefExtensionTests(Xunit.Abstractions.ITestOutputHelper output)
+        {
+            ServiceManagemenet.Common.Models.XunitTracingInterceptor.AddToContext(new ServiceManagemenet.Common.Models.XunitTracingInterceptor(output));
+        }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestSetChefExtensionBasic()
+        {
+            ComputeTestController.NewInstance.RunPsTest("Test-SetChefExtensionBasic");
```

Comment:
@NimishaS These tests run every day in CI and on people's boxes, so if there is an issue with module loading, it has to be in your local clone of the repo.  Please make sure you have updated to the latest, and done a clean rebuild.  AFter synicing,  ```git clean -xdf``` then rebuilding from scratch will clean out any artifacts from old builds.